extensions/spark/kyuubi-spark-authz/src/test/scala/org/apache/kyuubi/plugin/spark/authz/ranger/RangerSparkExtensionSuite.scala

```diff
@@ -388,6 +389,19 @@ abstract class RangerSparkExtensionSuite extends AnyFunSuite
           assert(sql(s"SELECT value1 FROM $db.${table}2").collect() ===
             Seq(Row(DigestUtils.md5Hex("1"))))
         })
+
+      doAs(
+        "admin",
+        sql(
+          s"INSERT INTO $db.$table SELECT 10, 1, 'hello', " +
+            s"'\u6d4b\u8bd5\u4e2d\u6587\u524d\u56db\u4e2aabcABC', " +
```

Comment:
Add comments to explain what and why to use this unicodes in tests. Source code is in UTF-8 encoding and good to have Chinese chars.

Reply:
This is because of Scala code style checks, which do not allow Chinese characters. Using Chinese characters may cause code style check errors, even though the code can still run.
![image](https://user-images.githubusercontent.com/1221600/229142592-d7fd0a70-b07e-460a-b753-d9e95d1922c1.png)

Reply:
No. Source code do support utf8 chars including Chinese. Also, you would prefer reformatting tool for lining and consider inline-method comments. Comments are meant to make the case readable and easy to understand, especially you don't have exact Chinese words in the sql.